docker/release/nanoserver/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# escape=`
```

Comment:
Oh, I see now. Would move this logic to `bootstrap.ps1` file (that's how it's done in [existing](https://github.com/PowerShell/PowerShell/blob/92b0eb336d7af74d5e439f58ffd3a367d511c0e5/docker/nightly/centos7/Dockerfile#L8) Dockerfiles) make sense?